app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/notifications/TaskyNotificationService.kt

```diff
@@ -0,0 +1,63 @@
+package com.davidtomas.taskyapp.features.agenda.data.notifications
+
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.core.app.NotificationCompat
+import com.davidtomas.taskyapp.R
+import com.davidtomas.taskyapp.features.agenda.domain.model.ScreenMode
+import com.davidtomas.taskyapp.features.agenda.presentation._common.navigation.AgendaRoutes
+
+class TaskyNotificationService {
```

Comment:
If a class doesn't require any constructor args and has no internal state, it could also just be an object